0.wave1_data/2.cellprofiler_analysis/scripts/loaddata_and_analysis.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Create LoadData csv with IC functions and run CellProfiler `analysis.cppipe` pipeline - Wave 1
+
+# ## Import libraries
+
+# In[1]:
+
+
+import os
```

Comment:
Was `os` used in this file? If not, consider removing this import.

Reply:
I have no idea why os just keeps appearing! It is fixed now.